mozart/poisson/fem/rectangle.py

```diff
@@ -0,0 +1,116 @@
+from sys import platform
+from os import path
+from ctypes import CDLL, c_double, c_void_p, c_int, c_bool
+import mozart as mz
+import numpy as np
+from mozart.common.etc import prefix_by_os
+dllpath = path.join(mz.__path__[0], prefix_by_os(platform) + '_' + 'libmozart.so')
+lib = CDLL(dllpath)
+
+from mozart.poisson.fem.common import nJacobiP, DnJacobiP
+
+def getMatrix(degree):
+	"""
+	Get FEM matrices on the reference domain I = [-1, 1]x[-1, 1]
+
+	Paramters
+		- ``degree`` (``int32``) : degree of polynomial
+
+	Returns
+		- ``M_R`` (``float64 array``) : Mass matrix on the reference domain
+		- ``Srr_R`` (``float64 array``) : Stiffness matrix w.r.t. rr on the reference domain
+		- ``Sss_R`` (``float64 array``) : Stiffness matrix w.r.t. ss on the reference domain
+		- ``Dr_R`` (``float64 array``) : Differentiation matrix w.r.t. r on the reference domain  
+		- ``Ds_R`` (``float64 array``) : Differentiation matrix w.r.t. s on the reference domain
+	"""
+
+	r = np.linspace(-1, 1, degree+1)
+	V = VandermondeM1D(degree, r)
+	invV = np.linalg.inv(V)
+	M_R = np.kron(np.dot(np.transpose(invV),invV),np.dot(np.transpose(invV),invV))
+	Dr_R = np.kron(np.eye(r.size),Dmatrix1D(degree, r, V))
+	Ds_R = np.kron(Dmatrix1D(degree, r, V),np.eye(r.size))
+	Srr_R = np.dot(np.dot(np.transpose(Dr_R),M_R),Dr_R)
+	Sss_R = np.dot(np.dot(np.transpose(Ds_R),M_R),Ds_R)
+	return (M_R, Srr_R, Sss_R, Dr_R, Ds_R)
+
+def VandermondeM1D(degree,r):
```

Comment:
It already exists. Use it. See this https://github.com/yoon-gu/Mozart/blob/master/mozart/poisson/fem/common.py#L154